### PR TITLE
Experiment: Phpcs with composer [ED-3182]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Comment by Nevoss** [comment]
@ronkelementor

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -31,14 +31,11 @@ jobs:
         uses: actions/checkout@master
       - name: Install Dependencies
         run: |
-          composer global require "squizlabs/php_codesniffer=*"
-          composer global require wp-coding-standards/wpcs
+          composer install
```
Comment (matipojo) [inline .github/workflows/lint.yml:34]: Can you install only the necessary packages for the linting?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (Nevoss) [reply .github/workflows/lint.yml:34]: cannot exclude packages. myabe the install should be in the build process and not in the lint proccess.

Comment (matipojo) [reply .github/workflows/lint.yml:34]: @ronkelementor please advise.

Comment (ronkelementor) [reply .github/workflows/lint.yml:34]: why do you want to exclude something?

Comment (Nevoss) [reply .github/workflows/lint.yml:34]: @matipojo @ronkelementor 
I think it is ok to install all the dependencies for the lint. 
we can later cache Composer's vendor folder, for now, it takes only a few seconds to install all the packages.

**Review by matipojo — APPROVED** [review]

**Pull request merged**